### PR TITLE
made simple db script to populate db if need to start over

### PR DESCRIPTION
**Pull request by NickWilsonDev**

### CODE REVIEW

File: db_util.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    PRIMARY KEY (userid),
+    cardid integer,
+    position integer,
+    CONSTRAINT cards_at_home_user_id_fkey FOREIGN KEY(userid)
+        REFERENCES users (ID) MATCH SIMPLE
+        ON UPDATE NO ACTION ON DELETE NO ACTION
+);
+
+
+CREATE TABLE all_cards (
```
Comment (rfpoulos) [inline db_util.sql]: If I ran this while it was empty, what would happen?  It has an open parentheses.

**Comment by rfpoulos** [comment]
Could you please add a "role" attribute to the user table as well?  Thanks.  After that, you have my approval to push to master.

File: db_util.sql [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+-- single line comment
+/* multi line comment */
+
+\c pc;
+
+CREATE TABLE users (
+    ID SERIAL PRIMARY KEY,
+    username VARCHAR (100),
+    password VARCHAR (100)
```
Comment (rfpoulos) [inline db_util.sql]: a "role" attribute would be really useful.

Comment (NickWilsonDev) [reply db_util.sql]: will do

**Pull request merged**